### ISSUE TITLE
What Activation, optimization functions of yolov5

### ISSUE THREAD

**Issue by moahaimen** [ultralytics/yolov5#9053]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

hi @glenn-jocher 
I writing a scientific paper and I need to be sure of the functions used in yolov5
so please if you could confirm with my information that:

Activation function = [Leaky ReLU ]
optimization function = [SGD]
 Cost Function =[Binary Cross-Entropy]
 Loss Function=[Logits Loss]

and also is learning rate=0.0001?
and Dropout?
thank you

**Comment by glenn-jocher** [comment]
@moahaimen hyperparameters like learning rate are in the hyp yaml file used for training, i.e. default is here:
https://github.com/ultralytics/yolov5/blob/f258cf8b37aeb3062230d43e1e9a4bf3b9874588/data/hyps/hyp.scratch-low.yaml#L1-L34

Activation function is SiLU

Losses are in utils/loss.py

There is no dropout
